src/devices/LidarLiteV3/category.txt

```diff
@@ -0,0 +1,2 @@
+lidar
```

Comment:
note: we'll ignore `lidar` category anyway for now since there is just 1 device there